virt/release_notes/virt-4-19-release-notes.adoc

```diff
@@ -63,6 +63,11 @@ This release adds new features and enhancements related to the following compone
 [id="virt-4-19-infrastructure"]
 === Infrastructure
 
+//CNV-49593
+* You can now prevent the inadvertent deletion of a virtual machine (VM) by xref:../../virt/managing_vms/virt-enabling-disabling-vm-delete-protection.adoc#virt-enabling-disabling-vm-delete-protection[enabling delete protection for the VM]. You can also disable delete protection that has been set for a VM.
++
+As a cluster administrator, you can xref:../../virt/managing_vms/virt-enabling-disabling-vm-delete-protection.adoc#virt-removing-vm-delete-protection_virt-enabling-disabling-vm-delete-protection[choose not to make the VM delete protection option available for any VMs].
```

Comment:
This is bit confusing so probably something like this : As a cluster administrator,To prevent users from enabling delete protection on virtual machines, you can disable the feature at the cluster level.

Reply:
@geetikakay (and @jcanocan), how about "As a cluster administrator, you can prevent users from enabling VM delete protection by disabling the option at the cluster level."

Reply:
I would say "feature" rather than "option". But just a personal preference.

Reply:
Lgtm

Reply:
Yeah  i missed feature